### PR TITLE
Backporting fixes

### PR DESCRIPTION
**Pull request by yhontyk**
### What does this PR do?
Backports context fixes from the downstreaming session.

### Specify the version of the product this PR applies to.

### CODE REVIEW

**Comment by themr0c** [comment]
I believe we should revert  only when there is an attribute in an attribute name "parent-context-of-blah". An attribute in the attribute value  is fine. 


As we can't use {prod-cli} here:
```
:parent-context-of-collecting-logs-using-che: {context}
```
As a consequence {prod-cli} can't be used here:

```
:context: {parent-context-of-collecting-logs-using-che}
```

This is not a big deal as parent-context-of-... never appears in a URL 

But this is still OK, and even necessary, because the {context} attribute is visible in URLs:

```
:context: collecting-logs-using-{prod-cli}
```

**Review by rkratky — APPROVED** [review]

File: src/main/pages/che-7/administration-guide/assembly_collecting-logs-using-chectl.adoc [hunk 1]
```diff
@@ -9,12 +9,12 @@ summary:
...
 
 :page-liquid:
-:parent-context-of-collecting-logs-using-{prod-cli}: {context}
+:parent-context-of-collecting-logs-using-chectl: {context}
 
 [id="collecting-logs-using-{prod-cli}_{context}"]
 = Collecting logs using {prod-cli}
 
-:context: collecting-logs-using-{prod-cli}
+:context: collecting-logs-using-chectl
```
Comment (rkratky) [inline src/main/pages/che-7/administration-guide/assembly_collecting-logs-using-chectl.adoc]:
```suggestion
:context: collecting-logs-using-{prod-cli}
```

File: src/main/pages/che-7/administration-guide/assembly_tracing-che.adoc [hunk 2]
```diff
@@ -9,12 +9,12 @@ summary:
...
 
 :page-liquid:
-:parent-context-of-tracing-{prod-id-short}: {context}
+:parent-context-of-tracing-che: {context}
 
 [id="tracing-{prod-id-short}_{context}"]
 = Tracing {prod-short}
 
-:context: tracing-{prod-id-short}
+:context: tracing-che
```
Comment (rkratky) [inline src/main/pages/che-7/administration-guide/assembly_tracing-che.adoc]:
```suggestion
:context: tracing-{prod-id-short}
```

File: src/main/pages/che-7/administration-guide/assembly_viewing-che-server-logs.adoc [hunk 3]
```diff
@@ -10,17 +10,17 @@ summary:
...
 // retrieving-{prod-id-short}-logs
 :page-liquid:
-:parent-context-of-viewing-{prod-id-short}-server-logs: {context}
+:parent-context-of-viewing-che-server-logs: {context}
 
 [id="viewing-{prod-id-short}-server-logs_{context}"]
 = Viewing {prod-short} server logs
 
-:context: viewing-{prod-id-short}-server-logs
+:context: viewing-che-server-logs
```
Comment (rkratky) [inline src/main/pages/che-7/administration-guide/assembly_viewing-che-server-logs.adoc]:
```suggestion
:context: viewing-{prod-id-short}-server-logs
```

**Review by rkratky — APPROVED** [review]
A few suggestions added.

**Review by themr0c — DISMISSED** [review]
The context attribute value should keep the reference to the {prod-cli} attribute as long as it is not referencing to a parent-context-something attribute.
```
:context: collecting-logs-using-{prod-cli}
```

**Review by themr0c — APPROVED** [review]

**Pull request merged**